src/analyses/abortUnless.ml

```diff
@@ -11,7 +11,7 @@ struct
 
   let name () = "abortUnless"
   module D = BoolDomain.MustBool
-  module C = Lattice.Unit
+  module C = Printable.Unit
 
   let context _ _ = ()
```

Comment:
Couldn't we have `include Analyses.UnitContextsSpec` to include just these two definitions without also being forced to include `IdentitySpec`?

Reply:
Yes, would be possible of course. The downside is that it would make many analyses a bit more verbose.

Reply:
I would be happy to make this change, in case we think the places where we can then use it outweighs the other places where it would make use more verbose.